libredex/ReachableClasses.cpp

```diff
@@ -9,6 +9,7 @@
 
 #include "ReachableClasses.h"
 
+#include <stdio.h>
```

Comment:
The use of `%d` and `\n` made me trigger the most relevant header file for this.